src/Xamarin.Android.Build.Tasks/Resources/MultiDexLoader.Debug.java

```diff
@@ -0,0 +1,512 @@
+//
+// This is a modified version of Bazel source code. Its copyright lines follow below.
+//
+
+// Copyright 2015 Xamarin Inc. All rights reserved.
```

Comment:
This copyright should be appended to, mentioning `Copyright 2017 Microsoft Corporation`.

Reply:
@jonpryor this was done. Typically the comment is hanging around because I didn't change this exact line :/